codecov.yml

```diff
@@ -3,5 +3,4 @@ codecov:
 ignore:
   - "contracts/test/"
   - "contracts/src/mock/"
-  - "contracts/src/libraries/"
-  - "contracts/src/state/"
\ No newline at end of file
+  - "contracts/src/state/operatorsRegistry/Operators.1.sol"
```

Comment:
@iamsahu Can you add a blank line EOF? By the way, we might want to add pre-commits for this 🤔 

Also, can you tell why we add this exclusion please?

Reply:
I will remove the exclusion I had added it to perform some checks.